tests/test_firewall.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from host import Host
```

Comment:
I mean that tests are failing https://travis-ci.org/rhevm-qe-automation/python-rrmngmnt/builds/262121251